doc/docs/master/concepts/data-concepts/commit.md

```diff
@@ -1,129 +1,92 @@
 # Commit
 
+!!! Note "Attention"
+         Note that Pachyderm uses the term `commit` at two different levels. A global level (check [GlobalID](../../advanced-concepts/globalID) for more details) and commits that occur in a repository. The following page details the latter. 
 ## Definition
 
 In Pachyderm, commits are atomic operations that **snapshot and preserve the state of
-the files and directories across repositories** at a point in time. A commit contains
-**sub-commits** in each repository it includes. These sub-commits are more like Git commits,
-recording the state of a single repository's file system.
-Unlike Git commits, however, Pachyderm sub-commits are centralized and transactional.
+the files and directories in a repository** at a point in time. 
+Unlike Git commits, Pachyderm commits are centralized and transactional.
 You can start a commit by running the `pachctl start commit` command with reference
-to a specific repository. At this point, sub-commits in all downstream repositories
-are also added to the commit, waiting on input data.
+to a specific repository. 
 After you're done making changes to the repository (`put file`, `delete file`, ...),
 you can finish your modifications by running the `pachctl finish commit` command.
-This command saves your changes and closes that repository's sub-commit,
+This command saves your changes and closes that repository's commit,
 indicating the data is ready for processing by downstream pipelines.
 
 !!! Warning
-    `start commit` can only be used on source repos (i.e. repos without [provenance](./provenance.md)).
+    `start commit` can only be used on input repos without [provenance](./provenance.md). Such repos are the entry points of a DAG.
     You cannot manually start a commit from a pipeline [output or meta repo](./repo.md).
 
-
-When you create a new commit, the **sub-commit at the head of each branch becomes the parent** of
-each corresponding sub-commit, by default containing the same repository state.
-Your repo history consists of those parent-child relationships between your data commits.
+ When you create a new commit, the previous commit on which the new commit is based becomes the parent of the new commit. Your repo history consists of those parent-child relationships between your data commits.
 
 !!! Note
-    An initial sub-commit has `<none>` as a parent.
+    An initial commit has `<none>` as a parent.
 
-Additionally, **sub-commits have an "origin"**.
-You can see an origin as the answer to: **"What triggered the production of this sub-commit"**.
+Additionally, **commits have an "origin"**.
+You can see an origin as the answer to: **"What triggered the production of this commit?"**.
 
 That origin can be of 3 types:
 
 - `USER`: The commit is the result of a user change (`put file`, `update pipeline`, `delete file`...)
-!!! Info
-    Every initial change is a `USER` change.
+    
+    !!! Info
+        Every initial change is a `USER` change.
```

Comment:
I guess I left this text in my version but I'm not sure what it means anymore - initial as in creating the branch

Reply:
Changed for: ```Every `USER` change is an initial commit.```